### PR TITLE
Add grpc-web-fake-transport Package

### PR DESCRIPTION
**Pull request by jonny-improbable**
Migrate FakeTransport form ts-protoc-gen package and add test coverage.

### CODE REVIEW

File: integration_test/ts/_proto/improbable/grpcweb/test/test_pb.js [hunk 1]
```diff
@@ -251,7 +251,7 @@ proto.improbable.grpcweb.test.PingRequest.prototype.getValue = function() {
 
 /** @param {string} value */
 proto.improbable.grpcweb.test.PingRequest.prototype.setValue = function(value) {
-  jspb.Message.setField(this, 1, value);
+  jspb.Message.setProto3StringField(this, 1, value);
```
Comment (easyCZ) [inline integration_test/ts/_proto/improbable/grpcweb/test/test_pb.js]: Seems to be unrelated to the PR. Generated with a different version of proto?

Comment (jonny-improbable) [reply integration_test/ts/_proto/improbable/grpcweb/test/test_pb.js]: confirmed generated with v1.5.0; we should, however, have better tooling around which version of protoc is used.
